AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/fanciful/FancyMessage.java

```diff
@@ -460,10 +461,29 @@ public void sendProposal(Player player, int order) {
 
 	private void sendProposal(CommandSender sender, String jsonString, int order) {
 		// ADDITION: Stops problems for Bedrock-connected players
-		if (!(sender instanceof Player) || sender.getName().startsWith("*") || sender.getName().startsWith(".")) {
+		/*
+		 * if floodgate is installed, we test if it is a floodgate player. This solves the problem of different prefixes.
+		 * We note, that this is more relyable than the previous method and solves any problem, beside the fact that the
+		 * sysadmin has to install floodgate on the backendservers in a bungeecord network. But this should be considered the easiest way.
+ 		 */
+
+		if(Bukkit.getServer().getPluginManager().getPlugin("floodgate")!=null){
```

Comment:
Maybe check if the plugin is also enabled? Could also separate this into a private method since line 501 also checks for Floodgate players.

Reply:
Changed it (View Commit 755b968)